build/release/container.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -e
+
+# Container builder
+# Must authenticate first: https://docs.github.com/en/packages/using-github-packages-with-your-projects-ecosystem/configuring-docker-for-use-with-github-packages#authenticating-to-github-packages
+# env vars: 
+#   $EARTHLY_BUILD_BRANCH: git branch to checkout
+#   $EARTHLY_BUILD_TAG: tag for container image
+
+
+EARTHLY_IMAGE_NAME="owncast"
+BUILD_TAG=${EARTHLY_BUILD_TAG:-webv2}
+DATE=$(date +"%Y%m%d")
+VERSION="${DATE}-${BUILD_TAG}"
+
+
+echo "Building container image ${EARTHLY_IMAGE_NAME}:${BUILD_TAG} ..."
+
+# Change to the root directory of the repository
+cd "$(git rev-parse --show-toplevel)" || exit
+if [ -n "${EARTHLY_BUILD_BRANCH}" ]; then
+    git checkout "${EARTHLY_BUILD_BRANCH}" || exit
+fi
+
+earthly --ci +docker-all --image="ghcr.io/owncast/${EARTHLY_IMAGE_NAME}" --tag="${BUILD_TAG}" --version="${VERSION}"
```

Comment:
Since you generalized this script to just `container.sh` it implies it's used to build all the container images, but that's not the case since Github is hard-coded here. Can the script maybe be renamed `nightlies.sh` or something to give a hint to people looking at it that it's not used for production builds and only used for nightly test builds?

Reply:
I renamed `build/release/container.sh` to `build/develop/container.sh`. I think this should make it clear enough. what do you think?

Reply:
Maybe a comment on top would help. I just am afraid somebody will want to make changes to the release docker image and start editing this script and I'll have to tell them the script isn't even used, so I'd rather make it super clear.

Reply:
I have the same concern with the Dockerfile vs. Earthfile.

Reply:
I added a note to the `Dockerfile`. Also made it clear in `build/develop/container.sh`.